ctapipe/image/extractor.py

```diff
@@ -27,6 +27,7 @@
 import numpy as np
 from numba import float32, float64, guvectorize, int64, njit, prange
 from scipy.ndimage import convolve1d
+from scipy.signal import filtfilt
```

Comment:
I found this in the scipy docs for this function:

> The function [sosfiltfilt](https://docs.scipy.org/doc/scipy/reference/generated/scipy.signal.sosfiltfilt.html#scipy.signal.sosfiltfilt) (and filter design using output='sos') should be preferred over [filtfilt](https://docs.scipy.org/doc/scipy/reference/generated/scipy.signal.filtfilt.html#scipy.signal.filtfilt) for most filtering tasks, as second-order sections have fewer numerical problems.

Is this applicable here?

Reply:
Not applicable to our simple filter, no. Thanks for checking though.